docs/getting-started/create-config-file.md

```diff
@@ -10,7 +10,7 @@ subsystem helps you collect data for a specific type of telemetry.
 
 - The **Prometheus** subsystem allows you collect metrics to send to Prometheus.
 - The **Loki** subsystem allows you to collect logs to send to Grafana Loki.
-- The **Tempo** subsystem allows you to collect spans to send to Grafana Tempo.
+- The **Traces** subsystem allows you to collect spans to send to Grafana Tempo.
```

Comment:
nit: can you change prometheus/loki to metrics/logs too? Sorry, I know it's a little out of scope :) If not, another PR can handle it